youtube_dl/extractor/vidlii.py

```diff
@@ -0,0 +1,136 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import re
+
+from .common import InfoExtractor
+from ..utils import (
+    int_or_none,
+    get_element_by_id, get_element_by_class, strip_or_none,
+    float_or_none, urljoin, js_to_json, unified_strdate)
+
+
+class VidliiIE(InfoExtractor):
+    _VALID_URL = r'(?:https*?:\/\/)*(?:www\.)*vidlii.com\/watch\?v=(?P<id>[^?\s]{11})'
```

Comment:
`[^?\s]` does not make any sense. You already matched `?` previously. I've already pointed out - no escapes for `/`.
`(?:https*?:\/\/)*` does not make any sense. `s*` does not make any sense. `(?:www\.)*` does not make any sense.

Reply:
FIXED.